src/components/side_panel/pivot/pivot_layout_configurator/pivot_measure/pivot_measure.xml

```diff
@@ -24,6 +24,7 @@
             composerContent="measure.computedBy.formula"
             defaultRangeSheetId="measure.computedBy.sheetId"
             contextualAutocomplete="getMeasureAutocomplete()"
+            invalid="isCalculatedMeasureInvalid"
```

Comment:
I'll let francois decide but it feels like the simple "o-invalid" doesn't draw enough attention